### DISCUSSION TITLE
How to compile v1 front-end code

### DISCUSSION THREAD

**Discussion by PterX** [firefly-iii/firefly-iii#8576] [Firefly III] [answered]
I am not a professional front-end programmer, but I want to implement some functions that are not currently in your plan. However, I noticed that the package.json in the root directory seems to be prepared for the v2 front-end. I want to know how to compile the vue code. Make my changes effective

**Comment by PterX** [comment]
@JC5

**Comment by JC5** [comment, marked as answer]
Look, I appreciate that you want to hack away in Firefly III, and I'll try to help you, but I think more than anything, you need to know that you can't just "add transaction types" and expect things to magically work as you expect them. Especially not using the v1 code.

But if you want, check out https://github.com/JC5/firefly-iii-dev/tree/main/v1-js for the v1 files.

**Reply by PterX** [reply]
Thank you very much for your help. Because it seems that v2 has a long cycle, I can only implement it automatically first. It seems that the URL you gave is what I want.
